reference/common/models/base64.v1.yaml

```diff
@@ -0,0 +1,4 @@
+title: Base64
+type: string
```

Comment:
We could use the `format` value to indicate base64 encoding of the `string` type: https://swagger.io/docs/specification/data-models/data-types/ - but I like the explicit pattern

Reply:
> We could use the `format` value to indicate base64 encoding of the `string` type: https://swagger.io/docs/specification/data-models/data-types/ - but I like the explicit pattern

I'm not against the explicit pattern at all, but I *would* like to see a $ref to his used in more places, so that I don't have to squint at the regexp in so many different places and try to double-check that there wasn't a cut and paste error made. :)